installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+
+You can deploy an installer-provisioned infrastructure cluster without specifying any customization to the cluster or its underlying machines to xref:../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[AWS], xref:../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[Azure], or xref:../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[GCP]. These installation methods are the fastest way to deploy a production-capable {product-title} cluster.
+
+If you need to perform basic configuration for your installer-provisioned infrastructure cluster, such as the region in your cloud that you deploy the machines to and the instance type for the cluster machines, you can customize an installation for xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[AWS], xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Azure], or xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[GCP].
+
+For installer-provisioned infrastructure installations, you can use an existing xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[VPC in AWS], xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[vNet in Azure], or xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[VPC in GCP]. You can also reuse part of your networking so that your cluster in xref:../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[AWS], xref:../installing/installing_azure/installing-azure-network-customizations.adoc#installing-azure-network-customizations[Azure], or xref:../installing/installing_gcp/installing-gcp-network-customizations.adoc#installing-gcp-network-customizations[GCP] can coexist with existing IP address allocations in your environment and integrate with existing MTU and VXLAN configurations. If you have existing accounts and credentials on these clouds, you can modify the accounts to have the required permissions to install {product-title} clusters on them.
+
+You can use the installer-provisioned infrastructure method to create appropriate machine instance in your data center for xref:../installing/installing_rhv/installing-rhv-default.adoc#installing-rhv-default[{rh-virtualization}], xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[vSphere], and xref:../installing/installing_bare_metal_ipi/ipi-install-overview#ipi-install-overview[bare metal].
+
```

Comment:
As a follow up to my earlier comment, this is where you could add the following dot title to help the reader scan this content:

`.Installing {product-title} by using user-provisioned infrastructure`